client-python/grakn/service/Session/TransactionService.py

```diff
@@ -0,0 +1,160 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import queue
+from typing import Type
+
+from grakn.service.Session.util.RequestBuilder import RequestBuilder
+import grakn.service.Session.util.ResponseConverter as ResponseConverter # for circular import issue
+from grakn.service.Session.util import enums
+from grakn.exception.ClientError import ClientError
+
+class TransactionService(object):
```

Comment:
Following the same logic in the feedback given for `KeyspaceService`, `TransactionService` should just be called `Transaction`. 

ps: Where's the `Session` class?

@flyingsilverfin @lolski

Reply:
Agreed to keep TransactionServiec

Session is in the __init__.py